docs/root/operations/cli.rst

```diff
@@ -226,12 +226,6 @@ following are the command line options that Envoy supports.
     option set to a non default value, you should use the same option (and same value) for subsequent hot
     restarts.
 
-.. option:: --max-stats <uint64_t>
```

Comment:
I think we actually need to deprecate this vs. outright remove, so there may be an upgrade race here in which people need to update with old CLI flags (potentially). So I would deprecate now and have setting this flag be a NOP and then we can remove in a future release.

Reply:
Done. I'm leaving just the TCLAP::ValueArg in; will that do it?

Reply:
Yes, but can you add this to deprecated.rst so that we track it for eventual removal?